resources/index-dev.html

```diff
@@ -17,7 +17,9 @@
 		<!-- VS Code Extensions Configuration -->
 		<meta id="vscode-workbench-builtin-extensions" data-settings="[]" />
 		<!-- Workbench Icon/Manifest/CSS -->
-		<link rel="icon" href="/favicon.ico" type="image/x-icon" />
+		<link rel="icon" media="(prefers-color-scheme:light)" href="/favicon.ico" type="image/x-icon">
+		<link rel="icon" media="(prefers-color-scheme:dark)" href="/favicon-light.ico" type="image/x-icon">
```

Comment:
Should `(prefers-color-scheme:light)` -> `/favicon-light.ico` and `(prefers-color-scheme:dark)` -> `/favicon.ico`?

Reply:
It works as expected, but it is confusing so will change the name of the favicons to fit their respective role.
`"(prefers-color-scheme:light)" href="/favicon-light.ico"` and `"(prefers-color-scheme:dark)" href="/favicon-dark.ico"`